ZuluContent/Zulu/Mobiles/CreatureProperties.cs

```diff
@@ -188,8 +189,13 @@ public static CreatureProperties Get(Type T)
             foreach (var (resistance, prop) in Resistances)
                 dest.Enchantments.SetResist(resistance, prop);
 
-            foreach (var (pack, amount) in Loot)
-                dest.AddLoot(pack, amount);
+            foreach (var (table, amount) in Loot)
```

Comment:
Not too sure what to do about this really... take it out entirely? Just changed it to be compat with the new interface

Reply:
Yep looks fine to me!